src/date-picker/date-picker-dialog.jsx

```diff
@@ -41,18 +42,6 @@ const DatePickerDialog = React.createClass({
     muiTheme: React.PropTypes.object,
   },
 
-  getDefaultProps: function() {
```

Comment:
It's an undocumented internal component, so that isn't technically a breaking change. I can put it back if you feel strongly about it.